sql/slave.h

```diff
@@ -32,16 +32,6 @@
   @file
 */
 
-/** 
-   Some of defines are need in parser even though replication is not 
-   compiled in (embedded).
-*/
-
-/**
-   The maximum is defined as (ULONG_MAX/1000) with 4 bytes ulong
-*/
-#define SLAVE_MAX_HEARTBEAT_PERIOD 4294967
```

Comment:
In the git commit message, it should mention how old conventions like this are changed in the code.

Reply:
Could you clarify what the old conventions were?

Reply:
In this case, it would be that metadata used for master info / relay log info fields were previously global macros/constants, but now have their definitions/scopes moved to these structs.